features/0360-use-did-key/README.md

```diff
@@ -45,7 +45,9 @@ Thus, `8HH5gYEeNc3z7PYXmd54d4x6qAfCNrqQqEB3nS7Zfu7K` becomes `did:key:z6MkmjY8Gn
 - Start with the original (presumably) base58 ed25519 public key
   - Since we have only a naked public key, we must assume the format
 - Base58 decode the public key to get the hex version
-- Multicodec: Prefix with the algorithm type. In this case, `0xed` is the key type of ed25519, per [multicodec table](https://github.com/multiformats/multicodec/blob/master/table.csv) (search for  `ed25519`).
+- Multicodec: Prefix with the varint of the algorithm type. In this case, that means prefixing the string with `varint(0xed)` (which is `0xed 0x01`) for the key type of ed25519,
```

Comment:
Note that it's an unsigned varint as defined here: https://github.com/multiformats/unsigned-varint

Reply:
Updated.